_data/download.yml

```diff
@@ -0,0 +1,138 @@
+- htmlsection: hardware 
+  title: Choosing Your Hardware 
+  links:
+    - url: /doc/system-requirements/
+      text: System Requirements
+      id: 1
+    - url: /doc/certified-hardware/
+      text: Certified Hardware
+      id: 2
+    - url: /hcl/
+      text: Hardware Compatibility List
+      id: 3
+
+- htmlsection: sec
+  title: Installing Qubes Securely 
+  links:
+    - url: /doc/installation-guide/
+      text: Installation Guide
+      id: 1
+    - url: /security/verifying-signatures/
+      text: How and Why to Verify Signatures
+      id: 2
+    - url: /doc/install-security/
+      text: Installation Security Considerations
+      id: 3
+
+- htmlsection: help
+  title: Help and Support 
+  links:
+    - url: /support/
+      text: Help and Support
+      id: 1
+    - url: /doc/#troubleshooting 
+      text: Troubleshooting Guides
+      id: 2
+    - url: /doc/reporting-bugs/
+      text: Report a Bug
+      id: 3
+
+- htmlsection: sec-info
+  title: Security Information 
+  links:
+    - url: /security/
+      text: Security Center
+      id: 1
+    - url: /security/pack/
+      text: Security Pack
+      id: 2
+    - url: /doc/security-guidelines/
+      text: Security Guidelines
+      id: 3
+
+- htmlsection: version-info
+  title: Version Information 
+  links:
+    - url: /doc/supported-versions/
+      text: Supported Versions 
+      id: 1
+    - url: /doc/templates/
+      text: Templates
+      id: 2
+    - url: /doc/version-scheme/
+      text: Version Scheme
+      id: 3
+
+- htmlsection: source-code
+  title: Source Code
+  links:
+    - url: /doc/source-code/
+      text: Source Code
+      id: 1
+    - url: /doc/coding-style/
+      text: Coding Guidelines
+      id: 2
+    - url: /doc/license/
+      text: Software License
+      id: 3
+
+- htmlsection: download-mirrors
+  title: Download Mirrors
+  table:
+    - column: Organization
+    - column: Location
+    - column: URL
+  links:
+    - url: /downloads/mirrors/#instructions-for-mirror-operators
+      text: How to contribute a download mirror  
+
+- htmlsection: donation 
+  title: Qubes OS is made possible by your donations!
+  paragraph: As a free and open-source software project, we rely on donations from users like you in order to keep running. Your contributions directly support the developers who work hard every day to improve your security. Please consider making a donation today.
+  url: /donate/
+  text: Donate
+
+- htmlsection: loop-releases
+  releases:
+    - r_version: testing
+      snippets:
+        - snippet: This is a 
+          url: /doc/testing/
+        - snippet: testing release
+        - snippet: Please help us improve it by 
+          url: /doc/reporting-bugs/
+        - snippet: reporting any bugs you encounter.
+        - snippet: For important work, we recommend the latest stable release. 
```

Comment:
This looks kind of scary. What's going on with these snippets?

Reply:
Now I see your comment, sorry I wasn't checking emails :) Yeah, the thing with the snippets. I am not happy about it either. Why: because converted all translatable files to md, by either extracting html into includes or layouts & their respective strings into data files. (Why no html: transifex can break html things) The snippets are consequences of this, in order to preserve the original layout and not introduce any more changes. But it is a start and once we figure out if downloads.html (f.ex.) can be just written with markdown or other tricks like including html directly into the markdown file, I can easily make the change :)